app/components/DateInput.tsx

```diff
@@ -43,7 +49,19 @@ export function DateInput({
         value={dateToYearMonthDayHourMinuteString(date)}
         min={min ? dateToYearMonthDayHourMinuteString(min) : undefined}
         max={max ? dateToYearMonthDayHourMinuteString(max) : undefined}
-        onChange={(e) => setDate(new Date(e.target.value))}
+        onChange={(e) => {
+          const updatedDate = new Date(e.target.value);
+          setDate(updatedDate);
+
+          //TODO: fix ESLint implicitly any error here
+          setDatesInputParentState(current => current.map(obj => {
```

Comment:
For this line, this is what I see in VSCode:

![image](https://user-images.githubusercontent.com/15804376/198811466-ff92a010-4df0-41bf-9e2e-d333ba24ea9a.png)

I think this can be fixed once I define a React Prop type appropriately

Reply:
The reason is because you have defined setDatesInputParentState as any on line 25!

Reply:
Thanks Sendou! This was related to my other comment & your suggestion helped me out there, so this will be resolved in the new PR as well